backend/internal/controllers/book_test.go

```diff
@@ -0,0 +1,86 @@
+package controllers_test
+
+import (
+	"encoding/json"
+	"github.com/TritonSE/words-alive/internal/controllers"
+	"github.com/TritonSE/words-alive/internal/database"
+	"github.com/TritonSE/words-alive/internal/models"
+	"github.com/stretchr/testify/require"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+var conn = database.GetConnection()
+var r = controllers.GetRouter()
+
+// Set up database
+func TestMain(m *testing.M) {
+	database.Migrate("../../migrations")
+
+	_, _ = conn.Exec("TRUNCATE books")
+
+	os.Exit(m.Run())
+}
+
+// Test for the book list function
+func TestGetBooks(t *testing.T) {
+	// Populate database
+	conn.Exec("INSERT INTO books (title, author) values ('c','c1');")
+	conn.Exec("INSERT INTO books (title, author) values ('a','a1');")
+	conn.Exec("INSERT INTO books (title, author) values ('b','b1');")
+
+	// Open server
+	ts := httptest.NewServer(r)
+	defer ts.Close()
```

Comment:
Can you move these to TestMain? That way we can recycle the same test server for each module.